### ISSUE TITLE
This is not working perfectly in responsive.

### ISSUE THREAD

**Issue by Naren-hybreeder** [kamranahmedse/driver.js#324]
Hello,
I have the below code on my website. On the desktop, it's working perfectly but when I checked on mobile it's not working as expected.

I am getting horizontal scroll and sometimes when clicking on next then it's not going to the next tour.

```
<a href="javascript:void(0);" id="start-tour" class="driver-next-btn me-3">Next</a>
<script type="text/javascript">
    $(document).ready(function() { 
 

document.getElementById('start-tour')
        .addEventListener('click', function(event) {
    event.stopPropagation();

const driver = new Driver({
allowClose: false,  
    closeBtnText:'<img src="/wp-content/uploads/2022/03/close-circle-tour.png" alt="">',
    prevBtnText: 'Previous', 
    nextBtnText: 'Next',
    doneBtnText: 'Done', // Text on the final button
    animate: false
});
    // Define the steps for introduction
driver.defineSteps([
  {
    element: '#guide1',
    popover: {
      className: 'tour',
      title: 'Some text here',
      //description: 'Body of the popover',
      position: 'right'
    }
  },
  {
    element: '#guide2',
    popover: {
      className: 'tour',
      title: 'Title heading here 2',
      //description: 'Body of the popover',
      position: 'right'
    }
  },
    {
    element: '#guide3',
    popover: {
      className: 'tour',
      title: 'Title heading here 3',
      //description: 'Body of the popover',
      position: 'top-center'
    }
  },
    {
    element: '#guide4',
    popover: {
      className: 'tour',
      title: 'Title heading here 4',
      //description: 'Body of the popover',
      position: 'top-center'
    }
  },
    

]);
    // Start the introduction
driver.start();
    
    
});
    
  });
``
![Screenshot 2022-03-18 at 8 42 08 PM](https://user-images.githubusercontent.com/16729825/159617147-fe45dc26-6bae-44f1-a4dd-a5b536f6d4ff.png)
`

**Comment by harshptl176** [comment]
Same issue @Naren-hybreeder

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. This issue should no longer exist. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
